sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -183,14 +188,17 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
 
           PCollectionTuple outputTuple =
               kafkaValues.apply(
-                  ParDo.of(new ErrorFn("Kafka-read-error-counter", valueMapper))
+                  ParDo.of(new ErrorFn("Kafka-read-error-counter", valueMapper, failOnError))
                       .withOutputTags(OUTPUT_TAG, TupleTagList.of(ERROR_TAG)));
 
-          return PCollectionRowTuple.of(
-              "output",
-              outputTuple.get(OUTPUT_TAG).setRowSchema(beamSchema),
-              "errors",
-              outputTuple.get(ERROR_TAG).setRowSchema(ERROR_SCHEMA));
+          PCollectionRowTuple outputRows =  PCollectionRowTuple.of("output",
+              outputTuple.get(OUTPUT_TAG).setRowSchema(beamSchema));
+
+          PCollection<Row> errorOutput = outputTuple.get(ERROR_TAG).setRowSchema(ERROR_SCHEMA);
```

Comment:
Can we use `ErrorHandling.errorSchema(...)` here? (Likewise, below, `ErrorHandling.errorRecord(...)`?)

Reply:
Done.